pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -283,58 +283,58 @@ def _handle_accelerator_and_strategy(self) -> None:
                 f"Passing `Trainer(accelerator={self.distributed_backend!r})` has been deprecated"
                 f" in v1.5 and will be removed in v1.7. Use `Trainer(strategy={self.distributed_backend!r})` instead."
             )
-            if self.strategy is not None:
+            if self.strategy_flag is not None:
                 raise MisconfigurationException(
-                    f"You have passed `Trainer(strategy={self.strategy!r})` but have"
+                    f"You have passed `Trainer(strategy={self.strategy_flag!r})` but have"
                     f" also passed `Trainer(accelerator={self.distributed_backend!r})`."
-                    f" HINT: Use just `Trainer(strategy={self.strategy!r})` instead."
+                    f" HINT: Use just `Trainer(strategy={self.strategy_flag!r})` instead."
                 )
-        if self.strategy == _StrategyType.TPU_SPAWN:
+        if self.strategy_flag == _StrategyType.TPU_SPAWN:
             raise MisconfigurationException(
                 "`Trainer(strategy='tpu_spawn')` is not a valid strategy,"
                 " you can use `Trainer(strategy='ddp_spawn', accelerator='tpu')` instead."
             )
-        if self.strategy == _StrategyType.DDP_CPU:
+        if self.strategy_flag == _StrategyType.DDP_CPU:
             raise MisconfigurationException(
                 "`Trainer(strategy='ddp_cpu')` is not a valid strategy,"
                 " you can use `Trainer(strategy='ddp'|'ddp_spawn', accelerator='cpu')` instead."
             )
 
-    def _set_training_type_plugin(self) -> None:
-        if isinstance(self.strategy, str) and self.strategy in TrainingTypePluginsRegistry:
-            self._training_type_plugin = TrainingTypePluginsRegistry.get(self.strategy)
-        if isinstance(self.strategy, str):
-            self.set_distributed_mode(self.strategy)
-        elif isinstance(self.strategy, Strategy):
-            self._training_type_plugin = self.strategy
+    def _set_strategy(self) -> None:
```

Comment:
Careful, all these changes here are not directly related to the deprecation of the Trainer.ttp property. 
In the future, please remain super focused, especially when these rename-PRs become large very quickly.

Reply:
@awaelchli If we only deprecate but not update training.training_type_plugin reference to trainer.strategy everywhere, All tests will fail (like everywhere calls a trainer.fit()), or we will have to add pytest.raiseError()...
Any recommendations how to separate the deprecation and update reference?

Reply:
it is clear that his cannot be separated. my comment was about the highlighted part where other methods and local variables get renamed that are unrelated to the property.

Reply:
Ohh! I see, I saw variable names in accelerator_connector all get updated, will remove these changes. I thought I separated them, thank you for reminder.

Reply:
@awaelchli  All non-trainer.training_type_plugin name change has removed

Reply:
That's great, will merge it after all classes have been renamed